src/Chainweb/Pact/PactService/Checkpointer.hs

```diff
@@ -625,7 +632,7 @@ rewindToIncremental rewindLimit (Just (ParentHeader parent)) = do
                     -- transactions (withBatchIO).
                     let playChunk :: IORef BlockHeight -> BlockHeader -> Stream (Of BlockHeader) IO r -> IO (Of BlockHeader r)
                         playChunk heightRef cur s = withBatchIO runPact $ \runPactLocal -> S.foldM
-                            (\c x -> x <$ (runPactLocal (fastForward (ParentHeader c, x)) >> writeIORef heightRef (_blockHeight c)))
+                            (\c x -> x <$ (runPactLocal (fastForward ValidateHashesOnFastForward (ParentHeader c, x)) >> writeIORef heightRef (_blockHeight c)))
```

Comment:
This is only ever called with `ValidateHashesOnFastForward`. So why not drop the parameter?

Reply:
It's called with ValidateHashesOnFastForward when we're doing a replay (i.e. `SyncToBlock`) but with `DoNotValidateHashesOnFastForward` when we're doing a fast forward as part of a rewind (i.e. the blocks have already been validated)